third-party/CMakeLists.txt

```diff
@@ -98,6 +98,10 @@ else()
   set(DEPS_C_COMPILER "${CMAKE_C_COMPILER}")
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
+  set(DEPS_C_COMPILER "${DEPS_C_COMPILER} -isysroot${CMAKE_OSX_SYSROOT}")
```

Comment:
should I restrict this for running on Mojave only? theoretically(!), this should work on old osx  too)

Reply:
Ideally it is generally applicable, but we'll find out since Travis is running an older macOS. :)

Reply:
yep, the build is red :( `CMAKE_OSX_SYSROOT` is empty